pkgs/dart_services/dart_test.yaml

```diff
@@ -0,0 +1,6 @@
+# Configure test execution; documentation available at:
+# https://github.com/dart-lang/test/blob/master/pkgs/test/doc/configuration.md.
+
+# Some of the project_creator_test.dart tests don't like to run in parallel with
+# other tests.
```

Comment:
Hmm, this might be something we should investigate.

Reply:
Probably; I don't think it's necessary a serious underlying issue, and even w/ this the tests finish in ~30 seconds. As you say though, it would be good to investigate, esp. to rule out something serious.